src/main/resources/META-INF/plugin.xml

```diff
@@ -57,7 +57,7 @@
 
     <change-notes><![CDATA[
       <ul>
-        <li>6.6 - Synchronize issues and taint vulnerabilities from the appropriate branch in connected mode. Subscribe for server events to update active rules. Quick fixes for 15 C/C++ rules.</li>
+	      <li>6.6 - Synchronize issues and taint vulnerabilities from the appropriate branch in connected mode. Subscribe for server events to update active rules and settings. Quick fixes for 15 C/C++ and 23 JS/TS rules. Support TypeScript 4.5. Drop support for Node 10. One more rule for C#. Performance improvements in Rider.</li>
```

Comment:
I recently removed `and settings` from the sentence because it was finally left out of the scope. Maybe you had a conflict on this line ?

Reply:
Indeed, good catch!